app/services/otp_rate_limiter.rb

```diff
@@ -36,30 +31,21 @@ def lock_out_user
   end
 
   def increment
-    # DO NOT MEMOIZE
-    @entry = OtpRequestsTracker.atomic_increment(entry_for_current_phone.id)
```

Comment:
This appears to be the only place `atomic_increment` was used. We could remove it from the `OtpRequestsTracker` class?

Reply:
thanks, yep